test/_s3_test.py

```diff
@@ -39,20 +42,18 @@
     # https://github.com/spulec/moto/issues/29
     print('Skipping %s because moto does not install properly before '
           'python2.7' % __file__)
-    from luigi.mock import skip
-    mock_s3 = skip
+    skip_tests = True
 
 
-if sys.version_info[:2] == (3, 4):
-    # moto break stuff under python3.4
-    from luigi.mock import skip
-    mock_s3 = skip
+# moto break stuff under python3.4
+skip_tests = skip_tests or sys.version_info[:2] == (3, 4)
 
 
 AWS_ACCESS_KEY = "XXXXXXXXXXXXXXXXXXXX"
 AWS_SECRET_KEY = "XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
 
 
+@_skipIf(skip_tests)
```

Comment:
I don't understand this function. Can't you use a `unittest.skipIf` decorator instead?

```py
@unittest.skipIf(sys.version_info[:2] == (3, 4))
class TestS3Target(unittest.TestCase):
    # ...
```